scripts/install_third_party.py

```diff
@@ -325,10 +325,10 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
-def _install_third_party_libs():
```

Comment:
Stick to the convention of calling this `main`

Reply:
Done